src/cloud/components/ContentManager/index.tsx

```diff
@@ -185,7 +184,7 @@ const ContentManager = ({
               />
             )
           })}
-          {orderedDocs.length === 0 && <EmptyRow label='No Documents' />}
+          {orderedDocs.length === 0 && <EmptyDoc>No documents shared</EmptyDoc>}
```

Comment:
- `documents shared` is not a good string as this content manager is used everywhere, including folders, where documents don't have to be shared. Revert to using `No Documents`. If you want to see no documents shared only in the shared page then it will be 

```
<EmptyRow label={page === 'shared' ? 'No documents shared' : 'No Documents'} />}

```
- `EmptyDoc` is 90% a duplicate of EmptyRow, it is not necessary. If the only concern is to remove the border then do it this way: 


```
<EmptyRow label='No Documents' bordered={false} />}
```

in EmptyRow

```
import cc from 'classcat' 

interface EmptyRowProps {
  label: string
  bordered?: boolean
}

const EmptyRow = ({ label, bordered = true }: EmptyRowProps) => {
  return (
    <Container className={cc(['empty__row', bordered && 'empty__row--bordered'])>
      <p>{label}</p>
    </Container>
  )
}

export default EmptyRow

const Container = styled.div`
  height: 40px;
  display: flex;
  align-items: center;
  padding: 0 ${({ theme }) => theme.sizes.spaces.l}px;
  color: ${({ theme }) => theme.colors.text.subtle};

  p {
    padding-left: ${({ theme }) => theme.sizes.spaces.md}px;
  }

&.empty__row--bordered {
 border-bottom: 1px solid ${({ theme }) => theme.colors.border.second};
}
`

```